compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/Passes.td

```diff
@@ -98,4 +98,12 @@ def DecomposeAttention :
   ];
 }
 
+def ConvertAttentionToOnlineAttention :
+    InterfacePass<"iree-linalg-ext-convert-attention-to-online-attention",
+    "mlir::FunctionOpInterface"> {
+  let summary = "";
+  let constructor = "mlir::iree_compiler::IREE::LinalgExt::"
+                    "createConvertAttentionToOnlineAttentionPass()";
```

Comment:
Let TD generate the constructor? We have started migrating IREE to this already, e.g., all gpu passes were ported over a couple of months ago.

Reply:
I would like to, but I think it needs to be done for LinalgExt in one go. Last time I tried, I found out that you cannot do it just for one pass, and have to do it for all of them.